pkgs/development/python-modules/python-frontmatter/default.nix

```
@@ -11,20 +11,15 @@ python3Packages.buildPythonPackage rec {
     sha256 = "1iki3rcbg7zs93m3mgqzncybqgdcch25qpwy84iz96qq8pipfs6g";
   };
 
-  propagatedBuildInputs = with python3Packages; [
-    pyyaml
-    six
-  ];
+  propagatedBuildInputs = [ pyyaml six ];
 
-  checkInputs = with python3Packages; [
-    pytest
-  ];
+  checkInputs = [ pytest ];
```

```suggestion
  checkInputs = [ pytestCheckHook ];
```

Maybe a separate PR.